### PR TITLE
Filter out inherited wikimedia-ui-base variables

### PR DESCRIPTION
**Pull request by jakobw**
Same as https://github.com/wmde/wikit-css/pull/25

### CODE REVIEW

File: tokens/.style-dictionary/config.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	include: ['node_modules/wikimedia-ui-base/tokens.json'],
+	source: ['properties/**/*.json'],
+	platforms: {
+		scss: {
+			transformGroup: 'scss',
+			buildPath: 'dist/',
+			files: [{
+				destination: '_variables.scss',
+				format: 'scss/variables',
+				filter: 'removePrefixedVars'
```
Comment (wiese) [inline tokens/.style-dictionary/config.js:13]: `removePrefixedVars` sounds rather generic but by the time it gets registered it is highly opinionated already (looking for the wmf-prefix). Maybe we should reflect this fact in its name to make the config more self-explanatory.

```suggestion
				filter: 'removeBaseTokensByWMF'
```

Comment (wiese) [reply tokens/.style-dictionary/config.js:13]: (changed my mind about the name and edited)

File: tokens/.style-dictionary/config.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+					filter: 'removePrefixedVars',
+				} ],
+			},
+			json: {
+				transformGroup: 'web',
+				buildPath: 'dist/',
+				files: [ {
+					destination: 'index.json',
+					format: 'json',
+					filter: 'removePrefixedVars',
```
Comment (wiese) [inline tokens/.style-dictionary/config.js:41]: It appears([0]) we can even use a pointer to a named function so no need for magic strings.

[0]: https://amzn.github.io/style-dictionary/#/formats?id=filtering-tokens

Comment (jakobw) [reply tokens/.style-dictionary/config.js]: Oh great! Changed it and also moved it to a separate file.

**Pull request merged**